src/main/java/com/chargerrobotics/subsystems/LimelightSubsystem.java

```diff
@@ -94,9 +256,36 @@ public void periodic() {
 		x = tx.getDouble(0.0);
 		y = ty.getDouble(0.0);
 		v = tv.getDouble(0.0);
+		s = ts.getDouble(0.0);
+		p = pip.getDouble(0.0);
 		SmartDashboard.putNumber("LimelightX", x);
 		SmartDashboard.putNumber("LimelightY", y);
-		SmartDashboard.putNumber("Limelightdistance", distance());
-		
+		SmartDashboard.putNumber("LimelightS", s);
+		SmartDashboard.putNumber("LimelightPipeline", p);
+		if (distance() != null) {
```

Comment:
This method will mean that `distance()` gets called twice, meaning you're doing the same work multiple times. Here's an alternative:
```
Double dist = distance();
if (dist != null) {
    SmartDashboard...
} else {
    SmartDashboard...
}